CotEditor/Sources/WebDocumentViewController.swift

```diff
@@ -51,6 +51,8 @@ final class WebDocumentViewController: NSViewController {
 
             self.webView?.configuration.userContentController.addUserScript(script)
         #endif
+
+        self.webView?.setValue(false, forKey: "drawsBackground")
```

Comment:
`drawsBackground` is actually a kind of private property. So, I prefer not to use it.